### ISSUE TITLE
[NBug] Could not load file or assembly 'Microsoft.VisualStudio.Thr...

### ISSUE THREAD

**Issue by gusarov** [gitextensions/gitextensions#8476]
## Error Details
```
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=16.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'Microsoft.VisualStudio.Threading, Version=16.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at GitExtensions.AzureDevOpsCommitMessage.Plugin.Execute(GitUIEventArgs args)
   at GitUI.CommandsDialogs.FormBrowse.<>c__DisplayClass63_0.<RegisterPlugins>b__2(Object <p0>, EventArgs <p1>)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

```

## Additional information
this always happens when trying to commit, also when open plugin -> TFS

## Environment
- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.27.0.windows.1
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4220.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Duplicate of https://github.com/gitextensions/GitExtensions.AzureDevOpsCommitMessage/issues/1

**Comment by gusarov** [comment]
Guys, but I've verified this on a clean install, the 16.5.132.47195 is deployed by installer in a target folder. What do you mean I should update? Installer deploys both GitExtensions.exe.config and dll, right?

**Comment by gusarov** [comment]
ah, ok, the linked bug is not closed yet, ok

**Comment by pmiossec** [comment]
Which version of the plugin do you use?
it's working well for me. Did you tried to update to the version 1.1 like requested?

Ask for reopening if you still have the problem and are providing more information (like plugin version)

**Comment by gusarov** [comment]
But I'm talking about latest installer, think while the latest installer is shipped with this bug it should not be closed.
Honestly I just realized that there is such a thing as Plugin Manager, this is not obvious and nobody mentioned this in any of 3 issues.
One of them is still opened by you:
https://github.com/gitextensions/gitextensions/issues/8003

I've fixed this by binding redirect, just want to make sure it is shipped with proper plugin version for other users.
